layer/logged_data/system_metrics.py

```diff
@@ -193,9 +253,24 @@ def __enter__(self) -> None:
             logged_data_destination=logged_data_destination,
         )
 
-        # By default, we get metrics from psutil
-        if "LAYER_FABRIC" in os.environ:
-            self._metrics_collector = DockerMetricsCollector(self._logger)
+        path = os.path.abspath("/sys/fs/cgroup/")
+        p = [
+            p
+            for p in psutil.disk_partitions(all=True)
+            if p.mountpoint == path.__str__()
+        ]
+        cgroup_mount_type = ""
+        if len(p) == 1:
+            cgroup_mount_type = p[0].fstype
```

Comment:
Nit: Can we move that to a function with some explanation what is going on here? Then it can return an enum:

```
class CGroupsVersion(Enum):
    V1 = "v1"
    V2 = "v2"
    UNKNOWN = "unknown"
```